pages/about.tsx

```diff
@@ -23,6 +23,24 @@ export async function getStaticProps({ locale }: any) {
 const About: React.FC = () => {
   const { t } = useTranslation("about")
 
+  const heroHeader: string = t("heroHeader")
+  const heroSubheader: string = t("heroSubheader")
+
+  const growingTitle: string = t("growingTitle")
```

Comment:
I haven't really commented on this much, but for some reason 'growingTitle' really gets to me haha
I get that the section is about the growing interest in Japan, but I feel like calling it growing<X> isn't quite right? If the wording gets changed away from growing, this will lose even more meaning to maintainers. Maybe we could change it to something that more describes the function of the section? I'd like the names to be more like heroHeader, or bibleContent, which convey the purpose/function rather than the details of the contents

perhaps `introduction` or `context` (e.g, `introductionTitle`, `introductionBlurb1`...)?
and having it be an array for the text would be better in case of future changes to number of sections

(same for beneath section)